implementations/rust/ockam/ockam/src/abac/workers.rs

```diff
@@ -0,0 +1,100 @@
+use crate::{
+    abac::{AbacLocalInfo, AbacMetadata, ABAC_LOCAL_INFO_IDENTIFIER},
+    Any, Context, OckamMessage, Result, Route, Routed, Worker,
+};
+use ockam_core::compat::boxed::Box;
+use ockam_core::{Decodable, LocalMessage};
+
+/// An `AbacUnwrapperWorker` wraps any [`AbacLocalInfo`] attached to a
+/// [`LocalMessage`] in `OckamMessage` [`Metadata`] so that it can be
+/// recovered after being sent to another Ockam node.
+pub struct AbacWrapperWorker {
```

Comment:
We don't have to use it, but I can think of use-cases where it could be useful for customers. If you'd prefer I can move it to the ockam_examples/ section.

Reply:
Moved to `ockam_examples/example_projects/access_control`.